ReactApp/src/components/staging/Example/Example2.jsx

```diff
@@ -2,10 +2,9 @@ import React from "react";
 
 import withStyles from "@mui/styles/withStyles";
 
-import RedirectToLogIn from "React-Automation-Studio/components/SystemComponents/RedirectToLogin.js";
 
-import TraditionalLayout from "React-Automation-Studio/components/UI/Layout/ComposedLayouts/TraditionalLayout.js";
-import CustomTextInput from "components/customComponents/CustomTextInput";
+import TraditionalLayout from "React-Automation-Studio/components/UI/Layout/ComposedLayouts/TraditionalLayout";
+import CustomTextInput from "../../../components/customComponents/CustomTextInput";
```

Comment:
Wouldn't it be more readable if we use "src/components..." instead? I have to focus to decode so many "..".